src/interstitial/InterstitialPlayer.tsx

```diff
@@ -115,6 +115,11 @@ export type InterstitialPlayerProps = {
    * Enable focus-based keyboard interactivity. Default: true.
    */
   enableFocus?: boolean
+  /**
+   * If true, the start button is shown which has to be clicked
+   * for playback/countdown to begin. Default: true.
+   */
+  hasStartButton?: boolean
```

Comment:
`hasStartButton` I did not connect hasStartButton inmediately with option to show it , but it is aligned with rest of naming, so I think we keep it in line.